Source/WebCore/dom/ElementRareData.cpp

```diff
@@ -51,4 +53,21 @@ struct SameSizeAsElementRareData : NodeRareData {
 
 static_assert(sizeof(ElementRareData) == sizeof(SameSizeAsElementRareData), "ElementRareData should stay small");
 
+ElementRareData::ElementRareData()
```

Comment:
Hm... It's a bit concerning to out-of-line these functions but we'll see what the perf bots say.

Reply:
Of course, that would be better

I started with the refactoring and then stumbled on the need for the fix; I can just do the fix only